npm/vue/${PWD}/cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+// ***********************************************************
```

Comment:
Looks like this file was accidentally created?

Reply:
yes... oops